packages/styled-components/src/models/test/StyleSheetManager.test.tsx

```diff
@@ -358,12 +358,12 @@ describe('StyleSheetManager', () => {
     `);
 
     expect(wrapper.toJSON()).toMatchInlineSnapshot(`
-            <div
-              className="sc-a b"
-            >
-              Foo
-            </div>
-        `);
+      <div
+        className="sc-a-10 b"
+      >
+        Foo
+      </div>
+    `);
```

Comment:
We can nicely see the incremented class names working in unit tests, since a single test file will have the same collection, and tests components create the same component multiple times.